posts/2022-10-25-22.0.0.11.adoc

```diff
@@ -0,0 +1,440 @@
+
+[source, xml]
+----
+    <featureManager>
+        <feature>sessionCache-1.0</feature>
+    </featureManager>
+
+    <!-- 
+        The 3rd-party JCache provider library that Liberty will use to manage and connect to the cache.
+     -->
+    <library id="JCacheProviderLib">
+        <fileset dir="${shared.resource.dir}" includes="jcacheprovider.jar" />
+    </library>
+
+    <!-- 
+        Configure the JCache cache manager.
+     -->
+    <cacheManager id="CacheManager" uri="uri://someuri">
+        <properties prop1="value1" prop2="value2" />
+
+        <cachingProvider jCacheLibraryRef="JCacheProviderLib" />
+    </cacheManager>
+
+    <!--
+        Configure the HTTP session cache.
+     -->
+    <httpSessionCache cacheManagerRef="CacheManager" ... />
+----
+
+=== Configuring Multiple Caches
+
+When configuring multiple distributed caches, instead of nesting the `cacheManager` configuration element within the cache element, the cache element needs to refer to the cache manager via the `cacheRef` attribute.
+
+[source, xml]
+----
+    <featureManager>
+        <feature>appSecurity-3.0</feature>
+        <feature>sessionCache-1.0</feature>
+    </featureManager>
+
+    <!-- 
+        The 3rd-party JCache provider library that Liberty will use to manage and connect to the cache.
+     -->
+    <library id="JCacheProviderLib">
+        <fileset dir="${shared.resource.dir}" includes="jcacheprovider.jar" />
+    </library>
+
+    <!-- 
+        Configure the JCache cache manager.
+     -->
+    <cacheManager id="CacheManager" uri="uri://someuri">
+        <properties prop1="value1" prop2="value2" />
+
+        <cachingProvider jCacheLibraryRef="JCacheProviderLib" />
+    </cacheManager>
+
+    <!-- 
+        Configure the JCache cache instances.
+     -->
+    <cache id="AuthCache" name="AuthCache" cacheManagerRef="CacheManager" />
+    <cache id="LoggedOutCookieCache" name="LoggedOutCookieCache" cacheManagerRef="CacheManager" />
+
+    <!-- 
+        Configured the authentication cache, logged-out cookie cache and HTTP session cache.
+     -->
+    <authCache cacheRef="AuthCache" />
+    <webAppSecurity loggedoutCookieCacheRef="LoggedOutCookieCache" ... />
+    <httpSessionCache cacheManagerRef="CacheManager" ... />
+----
+
+To find out more, check out the link:https://openliberty.io/docs/latest/reference/config/authentication.html[authentication] and link:{url-prefix}/docs/latest/reference/config/authCache.html[authCache] elements enabled by the link:{url-prefix}/docs/latest/reference/feature/appSecurity.html[appSecurity] feature, as well as the link:{url-prefix}/docs/latest/reference/feature/sessionCache.html#_examples[JCache Session Persistence examples].
+
+
+[#bell]
+== Expose SPI interfaces as BELL services and inject properties into BELL services
```

Comment:
Suggestion:
Expose SPI interfaces as BELL services and inject properties into BELL services
-->
Implement SPI interfaces with BELL services and enable BELL services to receive configuration properties